Framework/CurveFitting/src/Algorithms/FitPowderDiffPeaks.cpp

```diff
@@ -3339,7 +3315,7 @@ bool observePeakParameters(Workspace2D_sptr dataws, size_t wsindex,
  * @param Y :: vector to get maximum value from
  * @return index of the maximum value
  */
-size_t findMaxValue(const MantidVec Y) {
+size_t findMaxValue(const std::vector<double> &Y) {
```

Comment:
Is this used for anything but `HistogramY`? If not you could change the argument type and avoid the `rawData()` call above.